pkgs/applications/editors/vscode/extensions/contextmapper.context-mapper-vscode-extension/default.nix

```diff
@@ -0,0 +1,33 @@
+{ graphviz
+, jre
+, lib
+, makeWrapper
+, vscode-utils
+}:
+
+vscode-utils.buildVscodeMarketplaceExtension rec {
+  mktplcRef = {
+    name = "context-mapper-vscode-extension";
+    publisher = "contextmapper";
+    version = "6.7.0";
+    sha256 = "sha256-vlDVqn1Je0eo5Nf2gyotSvhIa07tWCINe79RZSyMzcA=";
+  };
+
+  nativeBuildInputs = [
+    graphviz
```

Comment:
This seems unused

Reply:
Thx for the input, I moved it to `buildInputs`